lib/dynamo.js

```diff
@@ -2,6 +2,7 @@
 
 const Promise = require('bluebird');
 const AWS = require('aws-sdk');
+const cache = require('./cache');
```

Comment:
That ... breaks my brain a little bit! I guess it's the require cache that makes that possible eh?